build.gradle

```
@@ -59,6 +59,7 @@ dependencies {
 
     // test
     testImplementation 'org.junit.jupiter:junit-jupiter:5.7.0'
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
```

Should we stay consistent with the other lines?
```suggestion
    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
```

---

We follow gradle upgrade guide here
https://docs.gradle.org/current/userguide/upgrading_version_8.html#manually_declaring_dependencies